src/components/header/Header.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import Modal from "../modal/Modal";
+
+import Burger from "../../images/burger.svg";
+import Logo from "../../images/logo.svg";
+import Phone from "../../images/smartphone.svg";
+
+import styles from "./Header.module.scss";
+
+const Header = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      <div className={styles.wrap}>
+        <button
+          type="button"
+          className={styles.btn_menu}
+          onClick={() => setIsOpen(true)}
```

Comment:
bad practice, better to set a function -
const openMenu = () => setIsOpen(true) 

and then 
onClick={openMenu}

perfomance issue